src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -481,17 +512,33 @@ describe('Step Registration', () => {
     const activationKeyDropdown = await screen.findByRole('textbox', {
       name: 'Select activation key',
     });
-    await user.click(activationKeyDropdown);
+    await act(async () => {
+      await user.click(activationKeyDropdown);
+    });
     const activationKey = await screen.findByRole('option', {
       name: 'name0',
     });
-    await user.click(activationKey);
+    await act(async () => {
+      await user.click(activationKey);
+    });
     screen.getByDisplayValue('name0');
 
-    screen.getByRole('button', { name: /Next/ }).click();
-    screen.getByRole('button', { name: /Next/ }).click();
-    screen.getByRole('button', { name: /Next/ }).click();
-    screen.getByRole('button', { name: /Next/ }).click();
+    const n1 = screen.getByRole('button', { name: /Next/ });
```

Comment:
tl;dr nothing needs to be fixed here from my point of view, just wanted to share my thoughts

For a moment I was thinking if there should be some naming scheme that could be used for these constants so the names are consistent throughout the tests. But to be completely honest, I don't think that's a massive problem right now. 

I agree that the `act`s are mostly a quick patch to get us to the point where we have clean test outputs and then we can go from there. The names and `act`s themselves will be a part of the refactoring so for now I'm perfectly god with those.